src/sentry/api_gateway/proxy.py

```diff
@@ -0,0 +1,50 @@
+"""
+Utilities related to proxying a request to a region silo
+"""
+
+import requests as external_request
+from requests import Response as external_response
+from rest_framework.exceptions import NotFound
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.models.organization import Organization
+
+
+def get_org_region(slug: str) -> str:
+    from sentry.types.region import get_region_for_organization
+
+    org = Organization.objects.get(slug=slug)
+    region_info = get_region_for_organization(org)
+    return region_info.address
+
+
+def build_url(region: str, request: Request) -> str:
+    """Might need some changes to handle other format region address."""
+    return region + request.path
```

Comment:
@AniketDas-Tekky you could use this utility function to generate the region URL https://github.com/getsentry/sentry/blob/b8bf714a5dc1997f580a20fb21e770d31b886f0d/src/sentry/api/utils.py#L179-L184

Reply:
Looks like this function only returns the url for the silo it's called from